voter-dapp/src/drizzleOptions.js

```diff
@@ -1,8 +1,9 @@
+import DesignatedVotingFactory from "./contracts/DesignatedVotingFactory.json";
 import Voting from "./contracts/Voting.json";
 import VotingToken from "./contracts/VotingToken.json";
 
 const options = {
-  contracts: [Voting, VotingToken],
+  contracts: [DesignatedVotingFactory, Voting, VotingToken],
```

Comment:
nit: are we using the DesignatedVotingFactory?

Reply:
We use it in the Dashboard component to look up the voter's DesignatedVoting contract.

Reply:
Oh right, of course. Thanks.